mva/analysis.py

```diff
@@ -64,7 +64,7 @@ class Analysis(object):
 
     def __init__(self, year,
                  systematics=False,
-                 use_embedding=True,
```

Comment:
I'm ok with switching off the embedding by default but this will impact the usage of the cmd arg `--no-embedding`. So either we modify it too or we let the embedding on by default.